packages/utilities/psammead-assets/src/amp-boilerplate.jsx

```diff
@@ -21,10 +36,24 @@ export const AMP_CONSENT_JS = (
     src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"
   />
 );
-export const AMP_ANALYTICS_JS = (
```

Comment:
This isn't removed, just re-ordered to have the JS exports listed alphabetically.